File: python/30-40 arası örnekler/40.ideal_kilo_hesaplama.py
b=eval(input("Boyunuzu giriniz: "))

c=int(input("Cinsiyetiniz [1-erkek / 2-kadın]: "))

if (c==1):
    ik=50+2.3*(b/2.54-60)
else:
    ik=45+2.3*(b/2.54-60)
print("\nİdeal kilonuz (kg): %0.2f"%ik)

File: c/30-40 arası örnekler/40_ideal_kilo_hesaplama.c
#include <stdio.h>
#include <conio.h>
int main()
{
    int b, ik, c;
    printf("Boyunuzu giriniz: ");
    scanf("%d", &b);
    printf("Cinsiyetiniz [1-Erkek / 2-Kadin]: ");
    scanf("%d", &c);
    if (c == 1)
        ik = 50 + 2.3 * (b / 2.54 - 60);
    else
        ik = 45.5 + 2.3 * (b / 2.54 - 60);
    printf("\nIdeal kilonuz (kg): %d\n", ik);

    return 0;
}